src/ds/d500/d500-auto-calibration-handler.h

```diff
@@ -0,0 +1,66 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2024 Intel Corporation. All Rights Reserved.
+
+#pragma once
+
+#include "d500-private.h"
+#include <src/hw-monitor.h>
+
+
+namespace librealsense
+{
+    enum class d500_calibration_state
+    {
+        RS2_D500_CALIBRATION_STATE_IDLE = 0,
+        RS2_D500_CALIBRATION_STATE_PROCESS,
+        RS2_D500_CALIBRATION_STATE_DONE_SUCCESS,
```

Comment:
Is this "algo done"? Else what is the difference between done and complete?
How much time do we spend in "done failure" before moving to "complete"?

Reply:
done is reported when the flash copy finishes. Complete is reported right after it.
If the flash burn fails, done_failure is reported instead of done_success.